glitcherbot_docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM php:7.4-apache
```

Comment:
I would maybe just call the folder `docker` - no need for `glitcherbot_docker`

Reply:
It was called docker initially but if I leave as docker, it gets mixed up with other projects in Docker Dashboard.
To see what I mean, click on the Docker icon on the toolbar and then Dashboard

Reply:
Do you still want it renamed to docker?